pkg/cluster/create.go

```diff
@@ -22,7 +23,7 @@ func Create(
 	repoUrl,
 	repoBranch,
 	basePath,
-	clusterSpecName string,
+	clusterSpecName string, // empty for gen2
```

Comment:
Can we just remove the field? Or is it still there because gen1 and gen2 clusters can coexist?

Reply:
Yes, the two co-exist at this stage. We can deprecate / retire gen1 later.